docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,265 @@
 # Getting Started with F# in Visual Studio Code with Ionide
 
-> [!NOTE]
-This is still in-progress.
\ No newline at end of file
+You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.
+
+## Prerequisites
+
+F# 4.0 or higher must be installed on your machine to use Ionide.
+
+### Windows
+
+If you're on Windows, the easiest way to do that is to install [Visual Studio with the Visual F# Tools](getting-started-visual-studio.md#installing-the-visual-f#-tools).
```

Comment:
Basically, this is the easiest way to get all of the dependencies that are required for the F# compiler and tools, since it required additional pieces.  Note that this isn't the case for the .NET Core version of the compiler, however, F# via VSCode is only officially done with .NET Framework/Mono right now.